neuromation/cli/job.py

```diff
@@ -300,20 +301,29 @@ def job() -> None:
 async def exec(
     root: Root,
     job: str,
-    tty: bool,
+    no_tty: bool,
     no_key_check: bool,
     cmd: Sequence[str],
     timeout: float,
 ) -> None:
     """
     Execute command in a running job.
+
+    Examples:
+
+    # Provides a shell to the container:
+    neuro exec my-job /bin/bash
+
+    # Executes a single command in the container and returns the control:
+    neuro exec --no-tty my-job ls > ls_output.txt && echo "OK"
+    neuro exec -T my-job /bin/not-an-executable || echo "failed"
```

Comment:
Providing an example for executing non-existing command looks not very informative.